dashboard/src/containers/AppUpgradeContainer/AppUpgradeContainer.tsx

```diff
@@ -24,11 +24,10 @@ function mapStateToProps(
 ) {
   return {
     app: apps.selected,
-    appsIsFetching: apps.isFetching,
+    formIsReady: (!apps.isFetching) && (!charts.isFetching),
```

Comment:
in other places we simply call this `isFetching` but it's okay

Reply:
That must have been an outdated diff you were looking at? (I'd originally collapsed them into `formIsReady` but then after reading about moving to hooks instead, it made more sense to use `chartsIsFetching` and leave `appsIsFetching` as is - so updated to that. Somehow you've got the intermediate diff (from https://github.com/kubeapps/kubeapps/pull/1810/commits/c73221a50bb47f127b0467cf50df046b918823ce ) rather than the final diff. Ah - perhaps from email?

Reply:
no no, I meant that in other components, this property would be usually named `isFetching` (instead of adapting the name for each component).